build/build.sh

```diff
@@ -8,6 +8,8 @@ set -o pipefail
 add_git_data_to_version() {
   local WMCO_SEMVER=$1
   GIT_COMMIT=$(git rev-parse --short HEAD)
+  # The commit hash retreived varies in length, so we will standardize to length received from the operator image
+  GIT_COMMIT="${GIT_COMMIT:0:7}"
```

Comment:
`7cee9a5e` len of 8 when run locally
`7cee9a5` len of 7 when run in operator image

Reply:
This change should go in a different PR